### ISSUE TITLE
Get-VSTeamAgent doesn't return the ID of the agents

### ISSUE THREAD

**Issue by okamilobo** [MethodsAndPractices/vsteam#337]
Steps to reproduce
------------------

```powershell

Set-VSTeamAccount -Account https://dev.azure.com/<org> -Token <token with full access to org>
Get-VSTeamAgent -PoolId <pool  Id>

Agent ID needed for subsequent calls.  
```

Expected behavior
-----------------
A list of the agents registered in the pool with all of the columns including Id populated with values.  

Actual behavior
---------------
A list of the agents registered in the pool with all of the columns populated with values EXCEPT Id .  

Environment data
----------------
OS
- [ ] Windows 10 Enterprise Version 1903

- VSTEAM verison 6.4.8

Server
- [ ] Azure DevOps Service

empty

> Get-VSTeamAPIVersion

Name                           Value
----                           -----
Release                        5.1-preview
Version                        VSTS
MemberEntitlementManagement    5.1-preview
TaskGroups                     5.1-preview.1
DistributedTask                5.0-preview
Core                           5.0
Packaging                      5.1-preview
ServiceFabricEndpoint          5.0-preview
VariableGroups                 5.0-preview.1
Build                          5.1-preview
Git                            5.1-preview
ExtensionsManagement           5.1-preview
Graph                          5.1-preview
Policy                         5.1
Tfvc                           5.0

> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.18362.752
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.18362.752
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by okamilobo** [comment]
![No Id](https://user-images.githubusercontent.com/7999246/86404089-92012c00-bc74-11ea-80a9-ff3985938562.png)

**Comment by SebastianSchuetze** [comment]
Thanks! We will check.

**Comment by DarqueWarrior** [comment]
Looking at it now.

**Comment by DarqueWarrior** [comment]
will be fixed in 6.5.1. Thanks so much for reporting this.

**Comment by DarqueWarrior** [comment]
Fix will be in 6.5.1
